src/components/PageBanner/PageBanner.js

```diff
@@ -8,58 +8,64 @@ import {
 import getColorVariant from '../../helpers/getColorVariant';
 import style from './PageBanner.scss';
 
-const PageBanner = props => (
-  <div style={{ backgroundColor: getColorVariant(props.colorName, '020') }}>
-    <div className={style.container}>
-      <div className={style.textAndIndicator}>
-        <div className={style.titleText}>
-          <h1
-            id={ELEMENT_IDS.INITIAL_FOCUS_POINT_ON_PAGE}
-            className={style.title}
-          >
-            {props.title}
+const PageBanner = (props) => {
+  const indicatorCardColor = props.indicatorSubCategory.colorName
+    ? getColorVariant(props.indicatorSubCategory.colorName, '500')
+    : getColorVariant(props.colorName, '500');
 
-            {props.isAllCitiesCategory && (
-              <span className={style.subTitle}> All cities</span>
-            )}
-          </h1>
+  return (
+    <div style={{ backgroundColor: getColorVariant(props.colorName, '020') }}>
+      <div className={style.container}>
+        <div className={style.textAndIndicator}>
+          <div className={style.titleText}>
+            <h1
+              id={ELEMENT_IDS.INITIAL_FOCUS_POINT_ON_PAGE}
+              className={style.title}
+            >
+              {props.title}
 
-          <p className={style.description}>{props.description}</p>
+              {props.isAllCitiesCategory && (
+                <span className={style.subTitle}> All cities</span>
+              )}
+            </h1>
 
-          {props.isAllCitiesCategory && (
-            <div className={style.featuredTextWrapper}>
-              <span className={style.featuredText}>Featured indicators</span>
+            <p className={style.description}>{props.description}</p>
 
-              <span> Showing range</span>
-            </div>
-          )}
-        </div>
+            {props.isAllCitiesCategory && (
+              <div className={style.featuredTextWrapper}>
+                <span className={style.featuredText}>Featured indicators</span>
 
-        <div>
-          {props.indicatorSubCategory && (
-            <div className={style.heroSubCategoryWrapper}>
-              <span className={style.heroSubCategoryText}>
-                Key indicator
-              </span>
+                <span> Showing range</span>
+              </div>
+            )}
+          </div>
+
+          <div>
+            {props.indicatorSubCategory && (
```

Comment:
`{props.indicatorSubCategory && (` is redundant isn't it? This is required and will always be true, so can go.

Reply:
This is true